Jenkinsfile

```diff
@@ -141,6 +141,8 @@ int terraform(String action, String tfStateBucket, String project, String enviro
     variablesMap.put('project', project)
     variablesMap.put('environment', environment)
     variablesMap.put('tf_state_bucket',tfStateBucket)
+    variablesMap.put('docdb_master_user',DOCDB_MASTER_USER)
```

Comment:
wrong place, should be below the declaration of tfVariables

Reply:
What do you think now